pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -414,77 +417,28 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
-                      <TableCell>
-                        <Link icon={<GitHubLogo />} iconAlign="start">
-                          Componenten GitHub
-                        </Link>
-                      </TableCell>
-                      <TableCell>
-                        <Link icon={<CallIcon />} iconAlign="start">
-                          010 - 123 456 7
-                        </Link>
-                      </TableCell>
-
-                      <TableCell
-                        className={styles.details}
-                        onClick={() => navigate("/organizations/5b9e0b17-00ca-433c-961b-913270643e6d")}
-                      >
-                        <Link icon={<ArrowRightIcon />} iconAlign="start">
-                          {t("Details")}
-                        </Link>
-                      </TableCell>
-                    </TableRow>
-
-                    <TableRow>
-                      <TableHeader>Gemeente Waterland</TableHeader>
-                      <TableCell>
-                        <Link icon={<GitHubLogo />} iconAlign="start">
-                          Componenten GitHub
-                        </Link>
-                      </TableCell>
-                      <TableCell>
-                        <Link icon={<CallIcon />} iconAlign="start">
-                          030 - 123 456 7
-                        </Link>
-                      </TableCell>
-
-                      <TableCell
-                        className={styles.details}
-                        onClick={() => navigate("/organizations/5b9e0b17-00ca-433c-961b-913270643e6d")}
-                      >
-                        <Link icon={<ArrowRightIcon />} iconAlign="start">
-                          {t("Details")}
-                        </Link>
-                      </TableCell>
-                    </TableRow>
-                  </TableBody>
-                </Table>
+                {_getComponent.data?.usedBy[0] &&
+                  tempOrganization.map((organization) => (
+                    <OrganizationCard
+                      key={organization.id}
+                      title={{
+                        label: organization?.name,
+                        href: `#`,
+                      }}
+                      description={organization?.description}
+                      website={organization?.website}
+                      logo={organization?.logo}
+                      components={{
+                        owned: organization?.owns?.length.toString() ?? "0",
+                        supported: organization?.supports?.length.toString() ?? "0",
+                        used: organization?.uses?.length.toString() ?? "0",
+                      }}
+                      gitHub={organization?.github}
+                      gitLab={organization?.gitlab}
+                      type={organization?.type}
+                      layoutClassName={styles.organizationCardContainer}
+                    />
```

Comment:
When showing the Organizations we we'll need to wrap it in a card grid like we do on the components page.
![image](https://user-images.githubusercontent.com/43807324/196358702-f068d923-3fff-4872-b7f1-1b32276ae5ba.png)